frontend/src/workbench/Site.tsx

```diff
@@ -11,128 +11,119 @@
-import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import IconButton from '@material-ui/core/IconButton';
+import { makeStyles } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
+import Typography from '@material-ui/core/Typography';
 import React, { useState } from 'react';
 import { SiteProps } from './Site.types';
 
 const useSiteStyles = makeStyles((theme) => ({
-  site: {
-    display: 'flex',
-    flexDirection: 'column',
-  },
-  expanded: {
-    flexGrow: 1,
-    overflow: 'auto',
-  },
-  accordionDetailsRoot: {
-    display: 'block',
-    padding: '0px',
+  leftSite: {
+    display: 'grid',
+    gridTemplateColumns: '32px 1fr',
+    justifyItems: 'stretch',
+    alignItems: 'stretch',
   },
-}));
-
-const Accordion = withStyles({
-  root: {
+  rightSite: {
     display: 'grid',
-    gridTemplateColumns: 'minmax(0,1fr)',
-    gridTemplateRows: 'min-content minmax(0,1fr)',
-    border: '1px solid rgba(0, 0, 0, .125)',
-    boxShadow: 'none',
-    '&:not(:last-child)': {
-      borderBottom: '0px',
-    },
-    '&:before': {
-      display: 'none',
-    },
-    '&$expanded': {
-      margin: '0px',
-    },
+    gridTemplateColumns: '1fr 32px',
+    justifyItems: 'stretch',
+    alignItems: 'stretch',
   },
-  expanded: {},
-})(MuiAccordion);
-
-const AccordionSummary = withStyles({
-  root: {
-    borderBottom: '1px solid rgba(0, 0, 0, .125)',
-    marginBottom: -1,
-    '&$expanded': {
-      minHeight: 48,
-    },
+  viewSelector: {
+    display: 'flex',
+    flexDirection: 'column',
+    background: 'var(--blue-lagoon-lighten-60)',
   },
-  content: {
-    '&$expanded': {
-      margin: '0px',
-    },
+  viewIcon: {
+    padding: theme.spacing(1),
   },
-  expanded: {},
-})(MuiAccordionSummary);
-
-const StyledCollapse = withStyles({
-  entered: {
+  view: {
+    display: 'grid',
+    gridTemplateRows: 'min-content 1fr',
+    justifyItems: 'stretch',
+    alignItems: 'stretch',
     overflow: 'auto',
   },
-})(MuiCollapse);
-
-const CustomCollapse = (props: CollapseProps) => {
-  const { children, ...collapseProps } = props;
-
-  const handleEntering = (node: HTMLElement, isAppearing: boolean) => {
-    node.style.height = 'auto';
-  };
-
-  const handleExit = (node: HTMLElement) => {
-    node.style.height = 'auto';
-  };
-
-  return (
-    <StyledCollapse {...collapseProps} onEntering={handleEntering} onExit={handleExit} timeout={0}>
-      {children}
-    </StyledCollapse>
-  );
-};
+  viewHeader: {
+    display: 'flex',
+    flexDirection: 'row',
+    padding: theme.spacing(1),
+    columnGap: theme.spacing(1),
+    borderBottomColor: theme.palette.divider,
+    borderBottomWidth: '1px',
+    borderBottomStyle: 'solid',
+  },
+  viewContent: {},
+}));
 
-export const Site = ({ editingContextId, selection, setSelection, readOnly, contributions }: SiteProps) => {
+export const Site = ({ editingContextId, selection, setSelection, readOnly, side, contributions }: SiteProps) => {
```

Comment:
Could you add a unit test for this component, it should be quite simple to test it.